scripts/packages_exporter.py

```diff
@@ -345,9 +368,26 @@ async def get_exporter_data(
 
         return list(dict(results).values())
 
-    async def sign_repomd_xml(self, data):
-        endpoint = "sign-tasks/sync_sign_task/"
-        return await self.make_request("POST", endpoint, data=data)
+    async def sign_repomd_xml(self, path_to_file: str, key_id: str):
+        token = await self.get_sign_server_token()
+        headers = {'Authorization': f"Bearer {token}"}
+        endpoint = 'sign'
+        params = {'keyid': key_id}
+        files = {'file': open(path_to_file, 'rb')}
+        result = {'asc_content': None, 'error': None}
+        try:
+            response = await self.make_request(
+                'POST',
+                endpoint,
+                params=params,
+                data=files,
+                user_headers=headers,
+                send_to='sign_server',
+            )
```

Comment:
I'd prefer not to assign unnecessary variables to the `make_request` function
example:
```python3
try:
    response = await self.make_request(
        'POST',
        endpoint,
        params={'keyid': key_id},
        ...
        files={'file': Path(path_to_file).read_bytes()}
    )
...
```

Reply:
done